src/EditorFeatures/Test2/Copilot/RoslynProposalAdjusterTests.vb

```diff
@@ -281,12 +301,210 @@ class C
 ")
         End Function
 
+        <WpfFact>
+        Public Async Function TestCSharp_RequiresFormatting() As Task
```

Comment:
i didn't look closely.  are tehre tests that exercise all the different permutations of the 3 fixes (or, at least, a test that exercises all 3?).